src/fuzzy.rs

```diff
@@ -48,6 +45,10 @@ pub(crate) fn match_vec(plines: &Vec<&str>, s: &str) -> bool {
     true
 }
 
+fn match_line(p: &str, s: &str) -> bool {
+    (p.starts_with(WILDCARD) && s.ends_with(&p[WILDCARD.len()..])) || p == s
```

Comment:
And we ensure that `p` and `s` are trimmed before they enter `match_vec`, so we don't need to handle whitespace either side of the line?

Reply:
Correct. I should probably make that clear though.

Reply:
Yeah a doc comment on the function wouldn't hurt